Examples/SuperwallUIKitExample/SuperwallUIKitExample/Services/PaywallService.swift

```diff
@@ -23,8 +23,8 @@ final class PaywallService {
     )
   }
 
-  static func trackDeepLink(url: URL) {
-    Paywall.track(.deepLinkOpen(deepLinkUrl: url))
+  static func handleDeepLink(_ url: URL) {
```

Comment:
Is this meant to be a breaking change? Original method was called `trackDeepLink(url: URL)`

Reply:
This is in the UIKit sample app, not the SDK haha